uml/uml.ttl

```diff
@@ -0,0 +1,472 @@
+@prefix : <http://bioprotocols.org/paml#> .
+@prefix om: <http://www.ontology-of-units-of-measure.org/resource/om-2/> .
+@prefix opil: <http://bioprotocols.org/opil/v1#> . # this needs to go - sbol_factory issue #10
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sbol: <http://sbols.org/v3#> .
+@prefix uml: <http://bioprotocols.org/uml#> .
+@base <http://bioprotocols.org/uml#> .
+
+<http://bioprotocols.org/uml> rdf:type owl:Ontology ;
+                              owl:imports sbol:, opil:, om: ;
+                              rdfs:comment "Unified Modeling Languge (UML) subset, translated to an SBOL factory ontology." ;
+                              owl:versionInfo "1.0-alpha1" .
+
+#################################################################
+#    Annotation properties
+#################################################################
+
+###  http://www.w3.org/2002/07/owl#maxCardinality
+owl:maxCardinality rdf:type owl:AnnotationProperty .
+owl:minCardinality rdf:type owl:AnnotationProperty .
+
+opil:compositionalProperty rdf:type owl:AnnotationProperty . # should be SBOL - sbol_factory issue #10
+
+
+#################################################################
+#    Datatypes
+#################################################################
+
+###  http://www.w3.org/2001/XMLSchema#anySimpleType
+xsd:anySimpleType rdf:type rdfs:Datatype .
+#sbol:Identified rdf:type owl:Class .
+#sbol:TopLevel rdf:type owl:Class .
+
+
+#################################################################
+#    Behaviors
+#################################################################
+
+# Properties for TypedElement
+uml:type rdf:type owl:DatatypeProperty ;
+         rdfs:comment "For TypedElement abstract class; UML 2.5.1 specification section 7.5" ;
+         rdfs:domain sbol:Identified ; # should probably actually be a union
+         rdfs:range xsd:anyURI ;
+         rdfs:label "type" .
+
+# Properties for MultiplicityElement
```

Comment:
Am I right in thinking that `MultiplicityElement` has no existence in the ontology proper?  Can it appear as a URI in the `uml:type` property?  If so, should we be declaring its existence as an entity?

I'm not sure I properly understand what is and is not meta here.

Reply:
TypedElement and MultiplicityElement are abstract interfaces that require multiple inheritance that sbol_factory isn't set up to support at present.